sdk/monitor/Azure.Monitor.Query/api/Azure.Monitor.Query.netstandard2.0.cs

```diff
@@ -287,6 +305,22 @@ public partial class MetricResult
         public System.Collections.Generic.IReadOnlyList<Azure.Monitor.Query.Models.MetricTimeSeriesElement> TimeSeries { get { throw null; } }
         public Azure.Monitor.Query.Models.MetricUnit Unit { get { throw null; } }
     }
+    public partial class MetricResultsResponse
+    {
+        internal MetricResultsResponse() { }
+        public System.Collections.Generic.IReadOnlyList<Azure.Monitor.Query.Models.MetricResultsResponseValuesItem> Values { get { throw null; } }
+    }
+    public partial class MetricResultsResponseValuesItem
+    {
+        internal MetricResultsResponseValuesItem() { }
+        public string Endtime { get { throw null; } }
+        public System.TimeSpan? Interval { get { throw null; } }
+        public string Namespace { get { throw null; } }
+        public string Resourceid { get { throw null; } }
+        public string Resourceregion { get { throw null; } }
```

Comment:
Should this be an extensible enum? Casing should be ResourceRegion.

Reply:
leaving as string but fixing camelcase